app/build.gradle

```diff
@@ -217,7 +219,7 @@ android {
 
     buildTypes {
         release {
-            signingConfig signingConfigs.release
+            //signingConfig signingConfigs.release
```

Comment:
Hey @josikie .  Will you revert?  Per the notes in our chat to allow you to compile

Reply:
Hello! 
Do you mean to delete them? @kcw-grunt 
I commented them so I could compile. Don't worry, I already ran it after put the changes

Reply:
I'll uncomment